cdap-docs/examples-manual/source/examples/data-cleansing.rst

```diff
@@ -0,0 +1,170 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform DataCleansing Application
+    :copyright: Copyright © 2014 Cask Data, Inc.
+
+.. _examples-word-count:
+
+==============
+Data Cleansing
+==============
+
+A Cask Data Application Platform (CDAP) example demonstrating incrementally consuming partitions of a partitioned file
+set using MapReduce.
```

Comment:
I am sure I agree. PartitionedFileSet either decomposes to partitioned file set or remains compound.